frontend/src/components/managePermissions/table/DelegatedAdminTable.vue

```diff
@@ -172,6 +158,18 @@ const highlightNewDelegatedAdminAccessRow = (rowData: any) => {
                     header="Domain"
                     sortable
                 ></Column>
+                <Column field="user.first_name" header="Full Name" sortable>
+                    <template #body="{ data }">
+                        {{
+                            data.user.first_name && data.user.last_name
```

Comment:
since we haven't done the one time load, the first and last name could be none, in this case will display nothing.

Reply:
Sure, make sense.